src/auto-advance.js

```diff
@@ -50,11 +52,14 @@ const setup = (player, delay) => {
   // Before queuing up new auto-advance behavior, check if `seconds` was
   // called with a valid value.
   if (!validSeconds(delay)) {
+    player.playlist.autoadvance_.delay = null;
```

Comment:
I also wonder if we should warn something here.

Reply:
The problem with that would be that you would get a warning when turning off auto-advance with a non-numeric value (e.g. `autoadvance(null)` or `autoadvance(false)`).